src/hashbang.html

```diff
@@ -54,11 +54,11 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
     <meta name="HandheldFriendly" content="true"/>
     <meta name="MobileOptimized" content="320"/>
-    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
-    <link href='/assets/local.css' rel='stylesheet' type='text/css'>
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:700" rel="stylesheet" type="text/css"> 
```

Comment:
Why is it being used literally nowhere: neither in CSS or HTML, not as 700, `bold` or `<b></b>` ?